src/overlays/actors/ovl_En_Bb/z_en_bb.c

```diff
@@ -631,38 +615,69 @@ s32 EnBb_OverrideLimbDraw(PlayState* play, s32 limbIndex, Gfx** dList, Vec3f* po
     return false;
 }
 
+/**
+ * This maps a given limb based on its limbIndex to its appropriate index
+ * in the bodyPartsPos/Velocity arrays. An index of -1 indicates that the
+ * limb is not part of the bodyParts arrays.
+ */
+static s8 sLimbToBodyParts[BUBBLE_LIMB_MAX] = {
+    BODYPART_NONE,     // BUBBLE_LIMB_NONE
+    BODYPART_NONE,     // BUBBLE_LIMB_ROOT
+    BODYPART_NONE,     // BUBBLE_LIMB_CRANIUM_ROOT
+    BODYPART_NONE,     // BUBBLE_LIMB_JAW_ROOT
+    BUBBLE_BODYPART_0, // BUBBLE_LIMB_JAW
+    BODYPART_NONE,     // BUBBLE_LIMB_LEFT_WING_ROOT
+    BODYPART_NONE,     // BUBBLE_LIMB_LEFT_WING_WRAPPER
+    BODYPART_NONE,     // BUBBLE_LIMB_LEFT_WING_WEBBING_ROOT
+    BUBBLE_BODYPART_1, // BUBBLE_LIMB_LEFT_WING_WEBBING
+    BODYPART_NONE,     // BUBBLE_LIMB_LEFT_WING_BONE
+    BODYPART_NONE,     // BUBBLE_LIMB_RIGHT_WING_ROOT
+    BODYPART_NONE,     // BUBBLE_LIMB_RIGHT_WING_WRAPPER
+    BODYPART_NONE,     // BUBBLE_LIMB_RIGHT_WING_WEBBING_ROOT
+    BUBBLE_BODYPART_2, // BUBBLE_LIMB_RIGHT_WING_WEBBING
+    BODYPART_NONE,     // BUBBLE_LIMB_RIGHT_WING_BONE
+    BUBBLE_BODYPART_3, // BUBBLE_LIMB_CRANIUM
+};
+
+/**
+ * The last element of the bodyParts arrays is a duplicate of the cranium
+ * limb, which is then offset by a certain amount. There is no display list
+ * associated with this, so it is only used for effects.
+ */
+static Vec3f sDuplicateCraniumBodyPartOffset = { 1000.0f, -700.0f, 0.0f };
```

Comment:
@tom-overton Since you documented this, but I'm wondering with the body part enum, we just name `BUBBLE_BODYPART_4` `BUBBLE_BODYPART_EFFECTS` and rename this to `sEffectsBodyPartOffset`?

Reply:
Haven't had a chance to look at this PR and see the full context, but just from what you're describing, that sounds fine with me.